app/modules/generateWallet.js

```diff
@@ -162,14 +163,16 @@ export const generateNewWalletAccount = (passphrase: string, passphrase2: string
         const { WIF, address } = account
         const encryptedWIF = wallet.encrypt(WIF, passphrase)
 
+        dispatch(saveAccountActions.call({label: walletName, address, key: encryptedWIF}))
+
         dispatch(hideNotification(infoNotificationId))
         dispatch(newWalletAccount({
           wif: WIF,
           address,
           passphrase,
           encryptedWIF
         }))
-        history.push(ROUTES.DISPLAY_WALLET_KEYS)
+        history.push(ROUTES.HOME)
```

Comment:
Sorry again for the confusion.  My original comment was just about importing wallets, and we don't need to show the data to the user in that use case.

For this use case, where the user is creating a new account, we do need to show the data for newly created accounts.  We don't necessarily have to redirect to a route to show this info off.  I'm happy to help figure this out.  We might need to split `saveAccountActions` into two separate actions -- one for creating the account, and one for saving it to local storage.